test/destroy_test_db

```diff
@@ -1,65 +0,0 @@
-#! /usr/bin/env bash
```

Comment:
This file was not used anywhere anymore. Does anyone still use it manually? I think this is handled by just recreating the whole database now, no?

Reply:
postgrest-docs still references it I think, but we will be able to significantly simplify those.

Reply:
Opened https://github.com/PostgREST/postgrest-docs/pull/378 to make sure we're not loosing track of that.